corehq/apps/reports/standard/monitoring.py

```diff
@@ -1355,7 +1355,7 @@ def _group_cell(self, group_id, group_name):
         url = self._make_url(base_url, params)
 
         return util.format_datatables_data(
-            self._html_anchor_tag(url, group_name),
+            self._html_anchor_tag(url, group_name.encode('utf-8')),
```

Comment:
I do think it's a better approach since it's generally not good to be passing bytes around in places where Unicode strings are expected.